src/BlazingPizza.Client/ServerAuthenticationStateProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Net.Http;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace BlazingPizza.Client
+{
+    public class ServerAuthenticationStateProvider : AuthenticationStateProvider
+    {
+        private readonly HttpClient http;
```

Comment:
Can we name the field the same as the parameter? Pick either.

Reply:
Ok, will do.